### PR TITLE
[Changelog] Update for Hammer-1

### PR DESCRIPTION
**Pull request by JPrause**
[skip ci]

### CODE REVIEW

**Comment by JPrause** [comment]
@miq-bot add_label documentation

**Comment by JPrause** [comment]
@miq-bot assign simaishi

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,21 +5,11 @@ All notable changes to this project will be documented in this file.
...
-
-### Fixed
-- Handle possibility that disk might not have sku [(#300)](https://github.com/ManageIQ/manageiq-providers-azure/pull/300)
-
-## Hammer Beta-1 - Released 2018-10-12
+## Hammer-1
 
 ### Added
 - Add plugin display name [(#283)](https://github.com/ManageIQ/manageiq-providers-azure/pull/283)
+- Add plugin display name [(#283)](https://github.com/ManageIQ/manageiq-providers-azure/pull/283)
```
Comment (simaishi) [inline CHANGELOG.md]: @JPrause duplicate

**Comment by simaishi** [comment]
Hammer backport details:

```
$ git log -1
commit a78589e0b2b1fc26231ddfe67be818ca70284b47
Author: Satoe Imaishi <simaishi@redhat.com>
Date:   Wed Jan 9 15:48:24 2019 -0500

    Merge pull request #320 from JPrause/hammer1_cl
    
    [Changelog] Update for Hammer-1
    
    (cherry picked from commit 434b56d555aab29b7553286345de36b3e4fb7b86)
```

**Pull request merged**